shuup/admin/static_src/base/js/form-utils.js

```diff
@@ -69,6 +69,43 @@ window.clearErrors = function ($form) {
     $form.parent().find(".errors").empty();
 };
 
+function inViewport (el) {
+
+    var r, html;
+    if ( !el || 1 !== el.nodeType ) { return false; }
+    html = document.documentElement;
+    r = el.getBoundingClientRect();
+
+    return ( !!r
+      && r.bottom >= 0
+      && r.right >= 0
+      && r.top <= html.clientHeight
+      && r.left <= html.clientWidth
+    );
+
+}
+
+window.onscroll = function() {
```

Comment:
add this event handler inside the block below to ensure all script were loaded before doing anything